source/data-encryption.txt

```diff
@@ -12,59 +12,167 @@ Data Encryption
+client as readable plaintext.
+
+.. figure:: /includes/images/queryable-encryption.svg
+   :figwidth: 750px
+   :alt: An image showing an example queryable encryption workflow.
 
 Recommendations
-~~~~~~~~~~~~~~~
+---------------
 
-For development and testing environments, do not enable added encryption with :ref:`your own encryption keys <security-kms-encryption>` through AWS Key Management Service (KMS), Google Cloud KMS, or Azure Key Vault. This saves costs in your 
-non-production environments.
+Consider the following security recommendations when 
+provisioning your {+clusters+}.
+
+BYOK Encryption
+~~~~~~~~~~~~~~~
 
-For staging and production environments, enable added encryption with :ref:`your own encryption keys <security-kms-encryption>` through AWS Key Management Service (KMS), Google Cloud KMS, or Azure Key Vault for
-additional security.
+For staging and production environments, enable :ref:`BYOK
+encryption <security-kms-encryption>` through AWS Key Management Service (KMS), 
+Google Cloud KMS, or Azure Key Vault when provisioning your {+clusters+} 
+to avoid relying on application development teams to configure it later on. 
+This also ensures consistent data protection across your environments.
+
+For development and testing environments, consider skipping |byok| encryption
+to save costs. However, if you're storing 
+sensitive data in |service|, such as for industries such as healthcare or 
+financial services, consider enabling |byok| encryption in development 
+and testing environments as well.
+
+Data Classification
+~~~~~~~~~~~~~~~~~~~
+
+During the provisioning process, we also recommend assessing the 
+sensitivity of certain fields in your data and classifying
+them to determine which data requires encryption and what global 
+restrictions to apply to these groups.
+
+Consider the following data classification levels as a guideline:
+
+- **Public Data**: Data that represents little to no
+  risk to the company if unauthorized disclosure, alteration, 
+  or destruction of data occurs. While confidentiality is 
+  less of a concern, you should still apply authorization controls to 
+  prevent unauthorized modification or destruction of public data.
+  
+  Examples: Products, Brochures, Training Information
+
+- **Private Data**: Data that represents a moderate
+  risk to the company if unauthorized disclosure, alteration, 
+  or destruction of data occurs. By default, all 
+  institutional data that is not explicitly classified as restricted or 
+  public data should be treated as private data. 
+  Apply CSFLE or queryable encryption on any fields that 
+  carry private data such as Personally identifiable information (PII).
+  
+  Examples: Customer Information, Contracts, Product Costs
+
+- **Restricted Data**: Data that represents significant
+  risk to the company if unauthorized disclosure, alteration, 
+  or destruction of data occurs. Apply the highest 
+  level of security controls to restricted data.
```

Comment:
Does this mean CSFLE or queryable encryption plus BYOK encryption? On all fields or just PII? Can we state that explicitly here?